ArduCopter/commands.cpp

```diff
@@ -106,7 +107,8 @@ bool Copter::set_home(const Location& loc, bool lock)
 bool Copter::far_from_EKF_origin(const Location& loc)
 {
     // check distance to EKF origin
-    const struct Location &ekf_origin = inertial_nav.get_origin();
+    struct Location ekf_origin;
+    ahrs.get_origin(ekf_origin);
```

Comment:
We should check the return of ahrs.get_origin here as well.  if we can't get it we should return false.